composer.json

```diff
@@ -22,19 +21,25 @@
             "email": "sebastian@spatie.be",
             "homepage": "https://spatie.be",
             "role": "Developer"
+        },
+        {
+            "name": "Tom Witkowski",
+            "email": "dev.gummibeer@gmail.com",
+            "homepage": "https://gummibeer.de",
+            "role": "Developer"
         }
     ],
     "require": {
         "php" : "^7.2",
-        "illuminate/config": "~5.8.0",
-        "illuminate/database": "~5.8.0",
-        "illuminate/support": "~5.8.0",
+        "illuminate/config": "^6.0",
```

Comment:
No more support for Laravel 5.8?

Reply:
Agreed - shouldn't be anything preventing support for both 5.8 or 6.0 with the same code base :+1: `5.8.*|^6.0` should suffice

Reply:
https://github.com/spatie/laravel-activitylog/pull/584/commits/6caffcfef81aef38941e2c0f810aee1d32fd7db1